docs/faq.rst

```diff
@@ -162,49 +162,43 @@ For example we have an application with two endpoints:
    2. ``/logout_user`` that when invoked needs to close all open
       websockets for that user.
 
-Keep in mind that you can only ``.close()`` a websocket from inside
-the handler task, and since the handler task is busy reading from the
-websocket, it can't react to other events.
-
-One simple solution is keeping a shared registry of websocket handler
-tasks for a user in the :class:`aiohttp.web.Application` instance and
-``cancel()`` them in ``/logout_user`` handler::
+One simple solution is keeping a shared registry of websocket responses
+for a user in the :class:`aiohttp.web.Application` instance and
+call :meth:`aiohttp.web.WebSocketResponse.close` on all of them in ``/logout_user`` handler::
 
     async def echo_handler(request):
 
-        ws = web.WebSocketResponse()
+        ws = aiohttp.web.WebSocketResponse()
```

Comment:
Please keep `web.WebSocketResponse` name.
Canonical way for using aiohttp web is `from aiohttp import web`.